app/services/vendor_api/single_application_presenter.rb

```diff
@@ -16,9 +15,9 @@ def as_json
           submitted_at: Time.now,
           personal_statement: 'hello',
           candidate: {
-            first_name: '',
-            last_name: '',
-            date_of_birth: '2019-01-01',
+            first_name: application_form.first_name,
+            last_name: application_form.last_name,
+            date_of_birth: application_form.date_of_birth,
```

Comment:
Just a question: If you were adding brand new JSON properties here would you need to manually update `config/vendor-api-0.5.0.yml`?

Reply:
Yes — and specs will fail if we don't, because we validate our responses against the schema in the YAML